simulators/blocktests/testrunner/Dockerfile

```diff
@@ -0,0 +1,16 @@
+# Docker container spec defining and running a simple smoke test consisting of
+# starting a single client node and retrieving some data from it.
+FROM jfloff/alpine-python:2.7-onbuild
+
+# Install curl to allow controlling the simulator as well as to interact with the
+# client via HTTP-RPC.
+RUN apk add --update curl git && \
+  git clone --depth 1 https://github.com/ethereum/tests.git
+
+
+# Add the simulation controller
+ADD simulator.sh /simulator.sh
+RUN chmod +x /simulator.sh
+COPY *.py /
+
+ENTRYPOINT ["python","/simulator.py"]
```

Comment:
The hierarchy is arbitrary, I wasn't sure how it should be structured. But to be specific, would you 
want me to replace

* simulators
    * blocktests
        * testrunner
            * Docker files etc

With 

* simulators
    * consensus
        * blocktests
            * Docker files etc

Or do you want me to skip a level (this is what I think you want): 
 
* simulators
    * consensus
        * Docker files etc